test/experimental/test_vectors.py

```diff
@@ -130,10 +133,6 @@ def test_errors(self):
         tokens = ['a', 'b', 'c']
         vecs = torch.stack((tensorA, tensorB,), 0)
 
-        with self.assertRaises(RuntimeError):
```

Comment:
So we are not covering this `assertRaises` anymore?

Reply:
No and this is because technically the `vectors` Tensor and the `tokens` can have different sizes since the Tensor can be sparse. This could occur when reading from a file because we initialize the tensor to be of the number of lines in the file. However, if there are duplicate tokens found in the file, we just skip the row corresponding to that line within the tensor.